scripts/uglify_ease_as_owl.py

```diff
@@ -0,0 +1,25 @@
+import subprocess
+import sys
+from pathlib import Path
+
+import owlready2
+
+
+def convert():
+    ease_ugly = Path(__file__).parent.parent / 'owl' / 'EASE-UGLY.owl'
+    if not ease_ugly.exists():
+        print('Generating rdf')
+        command = 'swipl -f prolog/uglify_ease.pl -t uglify_ease'.split()
+        proc = subprocess.run(command, cwd='data/ontologies/ease_ontology', capture_output=True)
```

Comment:
Is there any reason to still use the Prolog code at all? Couldn't we load all ontologies with owlready, and then export them?

It allows us to do transformation while loading.

Reply:
With owlready2, it would write a very short owl file which only features a few import statements, while the prolog code copies many triples directly, I don't know which one would be a better solution.